File: C++/مشق امشب باقر.cpp
#include <iostream>
using namespace std;
int main()
{
    int n1, n2, n3;
    cin >> n1 >> n2 >> n3;
    if ((n1 <= 180) && (n1 >= 1))
        if ((n2 <= 180) && (n2 >= 1))
            if ((n3 <= 180) && (n3 >= 1))
                if (n1 + n2 + n3 == 180)
                    cout << "Yes";
                else
                    cout << "No";
            else
                cout << "No";
        else
            cout << "No";
    else
        cout << "No";
}

File: Python/مشق امشب باقر.py
n1, n2, n3 = input().split()
n1 = int(n1)
n2 = int(n2)
n3 = int(n3)
if n1 in range(1, 180):
    if n2 in range(1, 180):
        if n3 in range(1, 180):
            num = n1+n2+n3
            if num == 180:
                print('Yes')
            else:
                print('No')
        else:
            print('No')
    else:
        print('No')
else:
    print('No')
